### DISCUSSION TITLE
What is the most ergonomic, frequently / quick feedback way to work on highlight / indent queries?

### DISCUSSION THREAD

**Discussion by NullVoxPopuli** [neovim/neovim#27393] [Q&A]
there are a number of tree-sitter-{lang} packages that I want to PR to, but I'm struggling with local development, and getting any of my changes within the local clones of these packgaes seen by my neovim.

What is the best way to develop highlights / indents / the other scm files?

**Comment by wookayin** [comment]
`:EditQuery` unless you are touching the parser (grammar).

Actually, `queries/lang/highlights.scm` will be load every time when the highlighter is attached, so `:edit <file>` on the buffer will also trigger reloading.
